src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -590,6 +592,56 @@ async def handle_list_tasks_command(
     await respond(text="Incident Task List", blocks=message, response_type="ephermeral")
 
 
+@app.action(
+    TaskNotificationActionIds.update_status,
+    middleware=[button_context_middleware, db_middleware],
+)
+async def handle_update_task_status_button_click(
```

Comment:
Feel free to rebase my changes and move it to the bottom of the file where all the code for the buttons is.